.github/workflows/ci.yml

```diff
@@ -0,0 +1,90 @@
+name: CI
+
+on:
+  pull_request:
+  push:
+    branches:
+      - master
+
+jobs:
+  qa:
+    name: QA
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        toxenv:
+          - checkqa
+          - readme
+    steps:
+      - uses: actions/checkout@master
+      - name: Set up Python ${{ matrix.python-version }}
+        uses: actions/setup-python@v2
+      - name: Set PY
+        run: echo "::set-env name=PY::$(python -VV | sha256sum | cut -d' ' -f1)"
+      - uses: actions/cache@v1
+        with:
+          path: ~/.cache/pre-commit
+          key: pre-commit|${{ env.PY }}|${{ hashFiles('.pre-commit-config.yaml') }}
+      - name: Install tox
+        run: pip install tox
+      - name: Test ${{ matrix.toxenv }}
```

Comment:
```suggestion
      - name: Prepare tox testenv
        run: tox --notest -p auto --parallel-live
      - name: Test ${{ matrix.toxenv }}
```

Reply:
pip-tools don't run parallel jobs with tox. What's the point with preparing testenv in a separate step?

Reply:
It's not about parallelism. It's about separating the virtualenv creation and deps installation from the actual tests run. First of all, it's easier to consume when you look at the separate steps and you can measure the time separately. But also, there were tox versions where the preparation step produced broken envs and still attempted to run tests which was awfully painful to debug. In some other CIs you they also have separate step names for install and script stages which fail and error out differently.

Reply:
This is a well-recognized technique nowadays: https://github.com/tox-dev/tox/issues/829#issuecomment-392290825 / https://github.com/tox-dev/tox/issues/1504#issuecomment-616083656 / https://github.com/tox-dev/azure-pipelines-template/blob/master/run-tox-env.yml#L133

Reply:
I see, good call.

Reply:
Could you elaborate on why there is `--parallel-live` option? Also, it's not clear why there is `-p auto`, since we run only a single envlist per job.

Reply:
oh, just power of habit

Reply:
I'll leave those params then in case we decide to run something in parallel. Thanks!  Addressed in 988dfbc.